catalogs/catalog-messaging-kafka/build.gradle.kts

```diff
@@ -14,6 +14,14 @@ dependencies {
   implementation(project(":api"))
   implementation(project(":core"))
   implementation(project(":common"))
+
+  implementation(libs.guava)
+
+  testImplementation(libs.junit.jupiter.api)
+  testImplementation(libs.mockito.core)
+  testImplementation(libs.hadoop2.common)
```

Comment:
Why do you need to hadoop dependency?

Reply:
Used for `tearDown` in UT :
https://github.com/datastrato/gravitino/blob/b42b8c94de4890e92582fdbcac35611c1f0ec9c3/catalogs/catalog-messaging-kafka/src/test/java/com/datastrato/gravitino/catalog/kafka/TestKafkaCatalogOperations.java#L94

Reply:
It's not necessary to introduce hadoop dependencies to only delete the folders. I think you can use the commons-io.